docs/pages/server-access/guides/jetbrains-sftp.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: JetBrains SFTP
+description: How to use JetBrains IDE to browse remote host with Teleport
+h1: SFTP with JetBrains IDE
+---
+
+JetBrain's IDEs, like PyCharm, Goland, Intellij, allow browsing, copying, and editing files on a remote server
+using SFTP protocol. The built-in integration will simplify the process if you are used to copying files back and forth
+using `scp` or other 3rd party clients. You can do the same things without leaving your IDE anymore.
```

Comment:
I think we can condense the final two sentences of this paragraph, maybe something like:

```markdown
You can integrate Teleport Server Access with your 
IDE so you can copy files to and from a remote 
machine without using a third-party client.
```